riverdriver/riverdatabasesql/migration/main/005_river_migration_add_line.down.sql

```diff
@@ -1,6 +1,22 @@
-DROP INDEX river_migration_line_version_idx;
-CREATE UNIQUE INDEX river_migration_version_idx ON river_migration USING btree(version);
+--
+-- If any non-main migration are present, 005 should be considered irreversible.
+--
```

Comment:
Any way we could make this intentionally error? Thinking of something like selecting from a nonexistent column `irreversible_migration` if any values are in the set of `SELECT FROM river_migration WHERE line != 'main'` or something

Reply:
Heh, yeah I thought about that too, but never willingly try to remember how to write PL/pgSQL ;)

Added.